tests/testthat/test-read.snapshot.R

```diff
@@ -0,0 +1,41 @@
+test_that("read.snapshot works", {
+  qe <- droplevels(quitteSort(as.quitte(quitte_example_dataAR6, na.rm = TRUE)))
+  tmpfile <- tempfile(pattern = "data", fileext = ".csv")
+  write.table(pivot_wider(qe, names_from = "period"), 
+            file = tmpfile, append = FALSE, quote = FALSE, sep = ",",
+            eol = "\n", na = "", dec = ".", row.names = FALSE,
+            col.names = TRUE) # mimick IIASA snapshot format
+  testlog <- tempfile(pattern = "log", fileext = ".txt")
+  testcommand <- c("grep 'odel'", "head -1", "tail -1", "sed 's/odel/ODEL/g;'")
+  for (t in testcommand) {
+    exitcode <- suppressWarnings(system(paste(t, tmpfile, ">", testlog),
+                                 ignore.stderr = TRUE, ignore.stdout = TRUE))
+    if (exitcode != 0) skip(paste0("read.snapshot tests skipped, as '", gsub(" .*", "", t),
+                                   "' is no available system command."))
+  }
```

Comment:
There is a proper way test for the availability of software tools, with the `--version` argument.
```
testcommand <- c('grep', 'head', 'tail', 'sed')

exitcodes <- suppressWarnings(
    sapply(paste(testcommand, '--version'), system,
           ignore.stdout = TRUE, ignore.stderr = TRUE))

if (any(0 != exitcodes)) {
    skip(paste("read.snapshot tests skipped, as",
               paste(paste0('`', testcommand[0 != exitcodes], '`'),
                     collapse = ', '),
               "are not available system commands."))
}
```

Reply:
Thanks. I moved that check into `read.snapshot` and catch this error in the tests in order to skip, if necessary.

Looks like that on Windows:
![image](https://github.com/pik-piam/quitte/assets/90761609/9cf5aa61-5ab2-4213-accf-904e14799390)